code/game/gamemodes/cult/cult_mode.dm

```diff
@@ -265,29 +265,56 @@ GLOBAL_LIST_EMPTY(all_cults)
+		cult_rise()
+		return
 
+	if(cult_players >= ascend_number)
+		cult_ascend()
+
+/datum/game_mode/proc/cult_rise()
+	cult_risen = TRUE
+	for(var/datum/mind/M in cult)
+		if(!M.current || !ishuman(M.current))
+			continue
+		SEND_SOUND(M.current, sound('sound/hallucinations/i_see_you2.ogg'))
+		to_chat(M.current, "<span class='cultlarge'>The veil weakens as your cult grows, your eyes begin to glow...</span>")
+		addtimer(CALLBACK(src, PROC_REF(rise), M.current), 20 SECONDS)
+
+
+/datum/game_mode/proc/cult_ascend()
+	cult_ascendant = TRUE
+	for(var/datum/mind/M in cult)
+		if(!M.current || !ishuman(M.current))
+			continue
+		SEND_SOUND(M.current, sound('sound/hallucinations/im_here1.ogg'))
+		to_chat(M.current, "<span class='cultlarge'>Your cult is ascendant and the red harvest approaches - you cannot hide your true nature for much longer!")
+		addtimer(CALLBACK(src, PROC_REF(ascend), M.current), 20 SECONDS)
+	GLOB.major_announcement.Announce("Picking up extradimensional activity related to the Cult of [SSticker.cultdat ? SSticker.cultdat.entity_name : "Nar'Sie"] from your station. Data suggests that about [ascend_percent * 100]% of the station has been converted. Security staff are authorized to use lethal force freely against cultists. Non-security staff should be prepared to defend themselves and their work areas from hostile cultists. Self defense permits non-security staff to use lethal force as a last resort, but non-security staff should be defending their work areas, not hunting down cultists. Dead crewmembers must be revived and deconverted once the situation is under control.", "Central Command Higher Dimensional Affairs", 'sound/AI/commandreport.ogg')
+
+/datum/game_mode/proc/cult_fall()
+	cult_ascendant = FALSE
+	for(var/datum/mind/M in cult)
+		if(!M.current || !ishuman(M.current))
+			continue
+		SEND_SOUND(M.current, sound('sound/hallucinations/wail.ogg'))
+		to_chat(M.current, "<span class='cultlarge'>The veil repairs itself, your power grows weaker...")
+		addtimer(CALLBACK(src, PROC_REF(descend), M.current), 20 SECONDS)
+	GLOB.major_announcement.Announce("Paranormal activity has returned to minimal levels. \
+									Security staff should minimize lethal force against cultists, using non-lethals where possible. \
+									All dead cultists should be taken to medbay or robotics for immediate revival and deconversion. \
+									Non-security staff may defend themselves, but should prioritize leaving any areas with cultists and reporting the cultists to security. \
+									Self defense permits non-security staff to use lethal force as a last resort. Hunting down cultists may make you liable for a manslaughter charge. \
+									All security gear that was handed out, should be returned. Access reset where needed, and weapons (including improvised) removed from the crew.",
```

Comment:
I think this could be re-worded a bit.

```suggestion
									Any access granted in response to the paranormal threat should be reset. Any and all security gear that was handed out should be returned. Finally, all weapons (including improvised) should be removed from the crew.",
```